.bemrc

```diff
@@ -1,22 +1,27 @@
-{
-    "root" : true,
-    "levels" : {
-        "./blocks" : {
-            "scheme" : "nested",
-            "naming" : "react",
-            "default" : true
+module.exports = {
+    root : true,
+    levels : {
+        blocks : {
+            scheme : 'nested',
+            shemeOptions : 'react',
```

Comment:
that's the name of the preset of `@bem/fs-scheme`.
see https://github.com/bem-sdk/bem-fs-scheme/blob/master/lib/presets/react.js